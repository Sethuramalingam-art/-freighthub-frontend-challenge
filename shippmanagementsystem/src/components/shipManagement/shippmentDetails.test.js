import React from 'react';
import { shallow } from 'enzyme';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import configureStore from 'redux-mock-store';
import ShippmentDetails from './shippmentDetails';

const mockStore = configureStore([]);
describe("Connected ShippmentDetails Component", () => {
    let store;
  let component;
  beforeEach(() => {
    store = mockStore({
    });
    component = renderer.create(
      <Provider store={store}>
        <ShippmentDetails />
      </Provider>
    );
  });
  it("should render ShippmentDetails component", () => {
    expect(component).toMatchSnapshot();
  });
});