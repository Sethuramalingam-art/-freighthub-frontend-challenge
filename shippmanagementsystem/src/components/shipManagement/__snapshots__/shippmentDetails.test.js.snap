// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Connected ShippmentDetails Component should render ShippmentDetails component 1`] = `undefined`;
